<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveWindow</class>
 <widget class="QWidget" name="SaveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>614</width>
        <height>454</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="intraLayout">
         <item>
          <widget class="QLabel" name="intraLabel">
           <property name="text">
            <string>Intra-speaker  Curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="intraComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="extraLayout">
         <item>
          <widget class="QLabel" name="extraLabel">
           <property name="text">
            <string>Extra-speaker  Curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="extraComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="GUI::DatabaseChart" name="chart" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="browserLayout">
         <item>
          <widget class="GUI::LineEditDropReceiver" name="browserLineEdit">
           <property name="placeholderText">
            <string>Choose destination file name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browserPushButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="saveLayout">
         <item>
          <spacer name="saveSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::LineEditDropReceiver</class>
   <extends>QLineEdit</extends>
   <header location="global">GUI/LineEditDropReceiver.hpp</header>
  </customwidget>
  <customwidget>
   <class>GUI::DatabaseChart</class>
   <extends>QWidget</extends>
   <header location="global">GUI/DatabaseChart.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
